src/test/java/com/atlassian/db/replica/it/example/aurora/AuroraClusterTest.java

```diff
@@ -31,8 +33,10 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class AuroraClusterTest {
-    final String writerEndpoint = "jdbc:postgresql://database-1.cluster-crmnlihjxqlm.eu-central-1.rds.amazonaws.com:5432/newdb";
-    final String readerEndpoint = "jdbc:postgresql://database-1.cluster-ro-crmnlihjxqlm.eu-central-1.rds.amazonaws.com:5432/newdb";
+    final AuroraJdbcUrl writerUrl = new AuroraJdbcUrl(
```

Comment:
Can it be simplified? (for example, created in one step if we never need the indirect AuroraEndpoint object)